testing/integration/tests/gas_calibration_test.rs

```diff
@@ -142,10 +140,7 @@ fn on_event_evm_shapes() {
     }
 
     for (obs, name) in vec![(validate_obs, CHARGE_VALIDATE), (accept_obs, CHARGE_ACCEPT)].iter() {
-        let regression = obs
-            .group_by(|a, b| a.label == b.label)
-            .map(|g| least_squares(g[0].label.to_owned(), g, 0))
-            .collect::<Vec<_>>();
+        let regression = run_linear_regression(&obs);
```

Comment:
Just curious, is this an opportunistic refactor or is it somehow related to switching to Rust stable?

Reply:
Its a required rewrite to get rid of the `#![feature(slice_group_by)]` which requires rust nightly to compile